common/cmake/v2x.cmake

```diff
@@ -155,57 +156,57 @@ function(VPR_TEST_PB_TYPE)
 
   set(DEPENDS_ARCH "")
   append_file_dependency(DEPENDS_ARCH "${symbiflow-arch-defs_SOURCE_DIR}/utils/template.arch.xml")
-  append_file_dependency(DEPENDS_ARCH "${VPR_TEST_PB_TYPE_TOP_MODULE}.pb_type.xml")
-  append_file_dependency(DEPENDS_ARCH "${VPR_TEST_PB_TYPE_TOP_MODULE}.model.xml")
+  append_file_dependency(DEPENDS_ARCH "${VPR_TEST_PB_TYPE_NAME}.pb_type.xml")
+  append_file_dependency(DEPENDS_ARCH "${VPR_TEST_PB_TYPE_NAME}.model.xml")
```

Comment:
Same comment here, NAME is no longer the correct parameter name.  Just use TOP_MODULE and forgo NAME if you are not going to respect the target naming.

Reply:
@kgugala